cysync/src/pages/mainApp/sidebar/wallet/addAccount/formStepComponents/Recipient.tsx

```diff
@@ -304,7 +304,8 @@ const Recipient: React.FC<StepComponentProps> = props => {
   const checkNearAccount = async (address: string) => {
     const coin = COINS[coinDetails.slug];
     if (coin instanceof NearCoinData) {
-      if (address.includes('.')) return 'This is not a valid Near address';
+      if (address.includes('.') || (address + nearSuffix).length > 64)
+        return 'This is not a valid Near address';
```

Comment:
Can we write the exact problem in the error msg? Something along the lines of `It should be less than 64 chars` or `It should end in .testnet`

Reply:
Sure